.kitchen.yml

```diff
@@ -31,7 +31,7 @@ platforms:
       box: windows-8.1
       box_url: http://cdn.box-cutter.com/windows/virtualbox4.3.20/eval-win81x64-enterprise-nocm-1.0.2.box
     provisioner:
-      require_chef_omnibus: 12.0.0
+      require_chef_omnibus: 12
```

Comment:
One downside of this I just came across - the PS script which [detects the currently installed](https://github.com/test-kitchen/test-kitchen/blob/windows-guest-support/support/chef_helpers.ps1#L6-L20) version only does direct string matching.  So putting a `12` here instead of `12.0.3` reinstalls the omnibus many times on an existing client.